vcd/resource_vcd_rde_type.go

```diff
@@ -16,6 +16,25 @@ import (
 	"github.com/vmware/go-vcloud-director/v2/util"
 )
 
+func getRdeTypeHookSchema(computed bool) *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"event": {
+				Type:        schema.TypeString,
+				Required:    !computed,
+				Computed:    computed,
+				Description: "Event that will invoke the Behavior, one of PostCreate, PostUpdate, PreDelete, PostDelete",
```

Comment:
What happens if a random string is passed here? Just checking if we should validate this field

Reply:
Well spotted! Added a validation function there.